Sources/Purchasing/Offering.swift

```diff
@@ -38,6 +38,13 @@ import Foundation
      */
     @objc public let serverDescription: String
 
+    private let _metadata: Metadata
+
+    /**
+     Offering metadata defined in RevenueCat dashboard.
+     */
+    @objc public var metadata: [String: Any] { self._metadata.data }
```

Comment:
I just realized we need to put this in `OfferingAPI.swift` and `RCOfferingAPI.m`